aggsender/db/aggsender_db_storage.go

```diff
@@ -7,7 +7,7 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/agglayer/aggkit/agglayer"
+	agglayerTypes "github.com/agglayer/aggkit/agglayer/types"
```

Comment:
Minor nitpick, AFAIK by convention import alias should be in lower case (although I agree it is less readable than the camel case)...

```suggestion
	agglayertypes "github.com/agglayer/aggkit/agglayer/types"
```

Reply:
Fix: https://github.com/agglayer/aggkit/pull/221/commits/3fcd74ae75e58b72998eb78d7fc454140a82e721